agreement/player_test.go

```diff
@@ -1802,6 +1802,118 @@ func TestPlayerRePropagatesFreshestBundle(t *testing.T) {
 	require.Truef(t, pM.getTrace().Contains(resynchEvent), "Player should try to repropagate freshest bundle")
 }
 
+func TestPlayerPropagatesProposalPayload(t *testing.T) {
```

Comment:
Can you please describe in comments what these tests are checking? 
Only one of the three fail without the fix.